tests/e2e/tests/test_api_methods.py

```diff
@@ -191,7 +191,7 @@ def test_workload_list(kiali_client):
 def test_workload_details(kiali_client):
     common_utils.get_response(kiali_client, method_name='workloadDetails', path={'namespace': 'bookinfo', 'workload':'details-v1'})
 
-def test_workload_health(kiali_client):
+def _test_workload_health(kiali_client):
```

Comment:
for removal, we need to add double underscore (__) right ?, here it has only one